src/fiatconnect/saga.ts

```diff
@@ -430,13 +444,13 @@ export function* _getSpecificQuote({
   cryptoAmount,
   flow,
   providerId,
-  fiatAccountType,
+  fiatAccount,
```

Comment:
could be nice to add a docstring on this function with a sentence on what it's for